aldryn_events/cms_appconfig.py

```diff
@@ -52,3 +52,11 @@ class EventsConfig(TranslatableModel, AppHookConfig):
     placeholder_events_registration_footer = PlaceholderField(
         'events_registration_footer',
         related_name='aldryn_events_registration_footer')
+
+    def __unicode__(self):
```

Comment:
Shouldn't this be def __str__(self) with the python_2_unicode_compatible decorator on the class?